CHANGELOG.md

```diff
@@ -28,7 +26,16 @@
 - Added support for `DatetimeIndex.mean` and `DatetimeIndex.std` methods.
 - Added support for `Resampler.asfreq`.
 - Added support for `resample` frequency `W`, `ME`, `YE` with `closed = "left"`.
-- Added support for file writes. This feature is currently in private preview.
+- Added support for string time-based `window` for `Rolling`.
+
+#### Improvements
+
+- Improved `to_pandas` to persist the original timezone offset for TIMESTAMP_TZ type.
+- Improved `dtype` results for TIMESTAMP_TZ type to show correct timezone offset.
+- Improved `dtype` results for TIMESTAMP_LTZ type to show correct timezone.
+- Improved error message when passing non-bool value to `numeric_only` for groupby aggregations.
+- Removed unnecessary warning about sort algorithm in `sort_values`.
+- Use SCOPED object for internal create temp tables. The SCOPED objects will be stored sproc scoped if created within stored sproc, otherwise will be session scoped, and the object will be automatically cleaned at the end of the scope.
```

Comment:
I think this still belongs to Snowpark Python not pandas.

Reply:
Based on this PR, I believe it's still a pandas feature: https://github.com/snowflakedb/snowpark-python/pull/2285